.setup/update_database.py

```diff
@@ -139,4 +139,10 @@
         os.system("""PGPASSWORD='{1}' psql --host={2} --username={3} --dbname={4} -c '{0}'""".format(create_table.format('course'), *variables))
         os.system("""PGPASSWORD='{1}' psql --host={2} --username={3} --dbname={4} -c "{0}" """.format(insert_row.format('course'), *variables))
         # add user/database
+
+        # add verify grader columns
+        os.system("""PGPASSWORD='{}' psql --host={} --username={} --dbname={} -c 'ALTER TABLE gradeable_component_data ADD COLUMN gcd_verifier_id VARCHAR(255)'""".format(*variables))
```

Comment:
I don't think we want to modify this file. I think this is all pre-migration code.